config/sync/core.date_format.hoeringsportal_today_date.yml

```diff
@@ -0,0 +1,8 @@
+uuid: e711cfb3-e492-4b08-ac79-2d529218db63
+langcode: da
+status: true
+dependencies: {  }
+id: hoeringsportal_today_date
```

Comment:
Should this format be called `hoeringsportal_day_only`? It has nothing to do with “today” …

Reply:
Absolutely. My bad.